### PR TITLE
Implemented SignOut From All Organization.

### PR DESCRIPTION
**Pull request by YenumJay**
Sign out from all organizations was implemented and this clears all stored data of the User from all organizations.

Sign out from the current organization was also implemented on the Profile page of the User which creates a better user experience for the user to easily sign out from the current organization.

https://user-images.githubusercontent.com/80979473/136666427-b044e419-4f64-4f1b-a2ae-768e00dbd78a.mp4

### CODE REVIEW

File: lib/ui/nav_pages/you_page/you_page_viewmodel.dart [hunk 1]
```diff
@@ -70,6 +80,33 @@ class YouPageViewModel extends BaseViewModel {
...
+      _snackBar.showCustomSnackBar(
+          message: "No internet connection, connect and try again.",
+          variant: SnackbarType.failure,
+          duration: const Duration(milliseconds: 1500));
+      return;
+    }
+
+    final response = await _apiService.post(endpoint, body: {}, token: token);
+
+    if (response?.statusCode == 200) {
```
Comment (pauleke65) [inline lib/ui/nav_pages/you_page/you_page_viewmodel.dart:101]: This should rather just sign out from the current organization and clear all route history while navigating back to the organisation view page

File: lib/utilities/internalization/localization/app_localization.dart [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    final path = 'assets/language/${locale.languageCode}.json';
+    final data = await rootBundle.loadString(path);
+    final Map<String, dynamic> _result = json.decode(data);
+
+    _sentences = <String, String>{};
+    _result.forEach((String key, dynamic value) {
+      _sentences[key] = value.toString();
+    });
+
+    // String jsonStringValues = await rootBundle
```
Comment (pauleke65) [inline lib/utilities/internalization/localization/app_localization.dart:30]: Could you remove this (unused code)

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Some Changes Requested

File: lib/ui/nav_pages/you_page/you_page_viewmodel.dart (hunk 1 above)
Comment (YenumJay) [reply lib/ui/nav_pages/you_page/you_page_viewmodel.dart:101]: Changes Implemented

File: lib/utilities/internalization/localization/app_localization.dart (hunk 2 above)
Comment (YenumJay) [reply lib/utilities/internalization/localization/app_localization.dart]: Unused Codes Removed

**Review by pauleke65 — APPROVED** [review]

**Pull request merged**